WebIDL/ecmascript-binding/class-string.any.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+test(() => {
+  assert_own_property(Blob.prototype, Symbol.toStringTag);
+
+  const propDesc = Object.getOwnPropertyDescriptor(Blob.prototype, Symbol.toStringTag);
+  assert_equals(propDesc.value, "Blob", "value");
+  assert_equals(propDesc.configurable, true, "configurable");
+  assert_equals(propDesc.enumerable, false, "enumerable");
+  assert_equals(propDesc.writable, false, "writable");
+}, "@@toStringTag exists on the prototype with the appropriate descriptor");
+
+test(() => {
+  assert_not_own_property(new Blob(), Symbol.toStringTag);
+}, "@@toStringTag must not exist on the instance");
+
+test(() => {
+  assert_equals(Object.prototype.toString.call(Blob.prototype), "[object Blob]");
+}, "Object.prototype.toString applied to the prototype");
+
+test(() => {
+  assert_equals(Object.prototype.toString.call(new Blob()), "[object Blob]");
+}, "Object.prototype.toString applied to an instance");
+
+test(t => {
+  assert_own_property(Blob.prototype, Symbol.toStringTag, "Precondition for this test: @@toStringTag on the prototype");
+
+  t.add_cleanup(() => {
+    Object.defineProperty(Blob.prototype, Symbol.toStringTag, { value: "Blob" });
```

Comment:
nit: It may be better to save the original property descriptor and restore it.

Historically Chrome implements IDL attributes as data properties for example.  This clean up might change the original settings.

Reply:
s/Chrome implements/Chrome implemented/

Chrome still implements cross origin attributes as data properties, though.

Reply:
I think it is OK, as if the test "@@toStringTag exists on the prototype with the appropriate descriptor" passes, then this will have the intended behavior.

Reply:
If you really expect the side effect (the original definition is gone and your new definition will be available), then maybe such code should be outside of "cleanup"?  I'd expect that "cleanup" restores the testing environment.

Anyway, this is a nit.